com.unity.cinemachine/Samples~/3D Samples/SplitScreenCar/SplitScreenCar.unity

```diff
@@ -0,0 +1,892 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
With the new input system, mouse controls one of the cars, but WASD controls both of the cars.  Can you fix this easily?  Might be tricky because we can only use the default input actions shipped with Input package.

Reply:
Fixed in latest commits